AbletonMCP_Remote_Script/handlers/session.py

```
@@ -0,0 +1,293 @@
+        raise
+
+
+def set_song_loop(song, enabled, start, length, ctrl=None):
+    """Control arrangement loop bracket."""
+    try:
+        if enabled is not None:
+            song.loop = bool(enabled)
+        if start is not None:
+            song.loop_start = max(0.0, float(start))
+        if length is not None:
+            l = float(length)
```

Ruff (0.14.14)

[error] Ambiguous variable name: `l`

(E741)